web/source/keymanweb.js

```diff
@@ -3089,39 +3087,36 @@
     var kbdLang = kbdStub['KL'];
     var kbdName = kbdStub['KN'];
 
-    if(typeof(kbdStub.asyncLoader.callback) == "function") {
-      // Add a handler for cases where the new <script> block fails to load.
-      Lscript.addEventListener('error', function() {
+
+    // Add a handler for cases where the new <script> block fails to load.
+    Lscript.addEventListener('error', function() {
+      if(kbdStub.asyncLoader.timer !== null) {
```

Comment:
If it were to be an issue, we'd also have it with the `load` event, as both are attached in parallel.